app/locales/en/translations.coffee

```diff
@@ -212,12 +212,14 @@ I18nTranslationsEn =
       "comments": "Description"
 
     "receive_package":
-      "inventory": "Inventory Number"
+      "inventory": "Print will generate number"
```

Comment:
@mcm-ham Can you add to zh/translations.coffee as well (just include english string) and then our translator will be able to edit the file directly in Github